ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/TransactionMeta.scala

```diff
@@ -19,11 +20,25 @@ import com.daml.lf.data.{Ref, Time}
   *   and to traffic-shape the work handled by DAML applications
   *   communicating over the ledger.
   *
+  * @param submissionTime: the transaction submission time
+  *
+  * @param submissionSeed: The seed used to derive the transaction contract IDs.
+  *
+  * @param optUsedPackages: the set of package IDs the transaction is depending on.
+  *   undefined is not known. Undefined is not known.
+  *
+  * @param optNodeSeeds: an association list that maps each ID of create and exercise nodes to
+  *   its respective seed. Undefined is not known.
+  *
+  * @param optByKeyNodes: the list of each ID of the fetch and exercise nodes that correspond
```

Comment:
IIUC you also have the lookupByKey nodes here

Reply:
Yes. For the sake of consistency all nodes that correspond to a "by-key" command are recorded here.

Reply:
Right, but the comment says that this contains only fetch and exercise nodes

Reply:
Indeed. Thanks.